client/js/components/statement/publicStatementModal/formGroups/FormGroupPostalAndCity.vue

```diff
@@ -12,6 +12,7 @@
     <div :class="prefixClass('layout')">
       <dp-input
         id="r_postalCode"
+        aria-labelledby="postalcode"
```

Comment:
thats not how it works. see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby

Reply:
then in this place we should use `aria-label` for screen renders and add a new prop in `DpInput`, right?
As I understood from the ticket that we add aria-label for all input fields, but I find all input fields have a label and are visible, but problem here only with PLZ field!

Reply:
I think it should resolved in demosplan-ui with a general approach